# cell type annotation papers

A place to collect papers that are related to cell-type annotation in single-cell RNA-sequence experiments
Really inspired by [Harold Pimentel's deep learning paper list](https://github.com/pimentel/deep_learning_papers). 

Ideally, this will be the basis for a collaborative, open review in the near future, such as the [Deep Review](https://github.com/greenelab/deep-review).

Contributions are welcome. 

Any papers that help answering the question "How do I label my cells in my single-cell RNA-seq analysis?" are within scope.

# Reviews / benchmarks

- [A comparison of automatic cell identification methods for single-cell RNA sequencing data](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1795-z)

# Methods

## Pre-print

- [CellMeSH: Probabilistic Cell-Type Identification Using Indexed Literature](https://www.biorxiv.org/content/10.1101/2020.05.29.124743v1)
- [Probabilistic gene expression signatures identify cell-types from single cell RNA-seq data(https://www.biorxiv.org/content/10.1101/2020.01.05.895441v2)

## Post-print
- [SciBet as a portable and fast single cell type identifier](https://www.nature.com/articles/s41467-020-15523-2)
- [scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data](https://www.cell.com/iscience/pdf/S2589-0042(20)30066-3.pdf)
- [SCSA: A Cell Type Annotation Tool for Single-Cell RNA-seq Data](https://www.frontiersin.org/articles/10.3389/fgene.2020.00490/full)
- [scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data](https://www.sciencedirect.com/science/article/pii/S2589004220300663)

# R package vignettes

- [Using SingleR to annotate single-cell RNA-seq data](https://bioconductor.org/packages/devel/bioc/vignettes/SingleR/inst/doc/SingleR.html)
- [OSCA - Bioconductor Cell Type Annotation](https://osca.bioconductor.org/cell-type-annotation.html)
